### PR TITLE
Clean up nuxt

### PR DESCRIPTION
**Pull request by Ray2126**
Clean up the Nuxt project to make better use of Vue 3 and Nuxt 3 best practices.

Test plan:
- Regression test that the app still behaves as expected.
- No new functionality as this is just a refactoring.

### CODE REVIEW

File: new-docs/components/Card.vue [hunk 1]
```diff
@@ -1,43 +1,29 @@
 <template>
-  <div
+  <NuxtLink
```
Comment (Ray2126) [inline new-docs/components/Card.vue:2]: https://v3.nuxtjs.org/api/components/nuxt-link

File: new-docs/components/Card.vue [hunk 2]
```diff
@@ -1,43 +1,29 @@
...
-    };
-  },
-
-  methods: {
-    goToLink() {
-      window.location = this.link;
-    },
-  },
-};
+<script setup>
```
Comment (Ray2126) [inline new-docs/components/Card.vue:21]: https://vuejs.org/api/sfc-script-setup.html

File: new-docs/components/Header.vue [hunk 3]
```diff
@@ -1,13 +1,8 @@
 <template>
-  <div class="w-full p-4 shadow-md">
+  <header class="w-full p-4 shadow-md">
     <div class="desktop-gutters flex items-center justify-center space-x-2">
-      <centrapay-logo class="w-12 h-12 fill-brand-accent" />
+      <lazy-icons-centrapay-logo class="w-12 h-12 fill-brand-accent" />
```
Comment (Ray2126) [inline new-docs/components/Header.vue:4]: https://v3.nuxtjs.org/guide/directory-structure/components#component-names and https://v3.nuxtjs.org/guide/directory-structure/components#dynamic-imports

File: new-docs/pages/v2.vue [hunk 4]
```diff
@@ -29,15 +33,5 @@
         link="https://docs.centrapay.com/api"
       />
     </div>
-  </div>
+  </main>
 </template>
-
-<script>
-export default {
```
Comment (Ray2126) [inline new-docs/pages/v2.vue:36]: This was unused.

File: new-docs/components/Card.vue (hunk 1 above)
Comment (AndyClifford) [reply new-docs/components/Card.vue:2]: 🔥

**Review by AndyClifford — APPROVED** [review]

**Pull request merged**